Update splice docs

waynecollier-da opened the pull request:
## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

tkuhrt commented on labs/splice.md:
```diff
@@ -1,22 +1,21 @@
-title: Splice
-parent: Labs
----
 # Splice
```
The above content is needed so that the lab shows up on https://labs.hyperledger.org

tkuhrt commented on labs/splice.md:
```diff
@@ -99,24 +103,14 @@ We expect that contributors interested in Splice may contribute, for example, ex
 The Validator module contains the Wallet module and the Traffic Acquisition module.
 
 The SV App module contains the Amulet smart contract code and the Amulet configuration variables, as well as the Synchronizer Governance app. 
-
-# Initial Committers
-
-- https://github.com/waynecollier-da
-- https://github.com/isegall-da
-
-# Sponsor
-https://github.com/tkuhrt - Tracy Kuhrt, Chair of the Hyperledger Technical Oversight Committee
-
-# Pre-existing repository
-We have an existing private repository. We plan to copy code from that repository to the new Splice repository in the first few weeks after the project has been approved, instead of transferring the repository itself.
```
These lines are part of the original template for lab proposals. Ideally we should probably keep these for history and to be consistent with the other lab proposals.

tkuhrt reviewed (commented):
We do not need to continue to have this file updated. I am okay with the updates, but if you would rather, it might be better to just link to the lab code where the README that you have with the lab continues to be updated. Let me know your thoughts.

waynecollier-da commented:
Hi, Tracy. Sorry for all the errors here.  I'l try again.

tkuhrt commented:
Looks like DCO was missed on the last two commits. Please follow the [instructions](https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/268/checks?check_run_id=25660645400) to fix.

waynecollier-da commented:
@tkuhrt FYI we cleaned up some of the language used in this lab description, and added a link to the Splice repo readme. After this update, we'll leave this lab description unchanged, and reference the readme for the Splice Lab repo itself.

waynecollier-da commented:
@tkuhrt Are we ok to merge this now?
